### PR TITLE
Public Announcement - June

### PR DESCRIPTION
**Pull request by anjai94**
Following security advisory will be added vi this PR.
- WSO2-2022-2177
- WSO2-2022-2023
- WSO2-2022-2101
- WSO2-2022-2182
- WSO2-2023-2447

### CODE REVIEW

File: en/docs/security-announcements/security-advisories/2023/WSO2-2022-2101.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+### SOLUTION
+If the latest version of the affected WSO2 product is not mentioned under the affected product list, you may migrate to the latest version to receive security fixes. Otherwise you may apply the relevant fixes to the product based on the public fix: 
+
+* [https://github.com/wso2-extensions/identity-governance/pull/616](https://github.com/wso2-extensions/identity-governance/pull/616)
+
+Further, If you are not using above mentioned behavior intentionally, please follow the below steps:
+
+* Apply the provided patch/update to the affected versions of the products.
+* Add the following configuration to deployment.toml file.
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2023/WSO2-2022-2101.md:44]:
```suggestion
* Add the following configuration to `deployment.toml` file.
```

File: en/docs/security-announcements/security-advisories/2023/WSO2-2022-2177.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+If you are an open-source user or using a product version that is EOL (End of License) :
+
+You may migrate to the latest version of the product if the latest version is not listed under the list of the affected products. Otherwise, you may apply the relevant fixes to the product based on the public fixes as given below:
+
+* [https://github.com/wso2-extensions/identity-carbon-auth-rest/pull/216](https://github.com/wso2-extensions/identity-carbon-auth-rest/pull/216)
+
+Or else you may follow the mitigation steps given below.
+
+#### Group 1:
+For below listed products, it is recommend to append the following configuration lines to the `&lt;PRODUCT-HOME&gt;/repository/conf/deployment.toml` file and restart the server.
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2023/WSO2-2022-2177.md:105]:
```suggestion
For below listed products, it is recommend to append the following configuration lines to the `<PRODUCT-HOME>/repository/conf/deployment.toml` file and restart the server.
```

File: en/docs/security-announcements/security-advisories/2023/WSO2-2022-2177.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+If you have already applied the above given configuration in your identity.xml, please add the additional `<Permissions>/</Permissions>` line as mentioned above & restart the server.
+
+
+### CREDITS
+WSO2 thanks, [Rick Roane](https://www.linkedin.com/in/rick-r/) for responsibly reporting the identified issue and working with us as we addressed it.
+
+
+### References
+[^1]: [https://is.docs.wso2.com/en/latest/guides/password-mgt/recover-password/ ](https://is.docs.wso2.com/en/latest/guides/password-mgt/recover-password/)
+[^2]: [https://apim.docs.wso2.com/en/latest/install-and-setup/setup/security/user-account-management/#enable-password-recovery ](https://apim.docs.wso2.com/en/latest/install-and-setup/setup/security/user-account-management/#enable-password-recovery)
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2023/WSO2-2022-2177.md:159]:
```suggestion
[^2]: [https://apim.docs.wso2.com/en/latest/install-and-setup/setup/security/user-account-management/#enable-password-recovery](https://apim.docs.wso2.com/en/latest/install-and-setup/setup/security/user-account-management/#enable-password-recovery)
```

File: en/docs/security-announcements/security-advisories/2023/WSO2-2022-2357.md [hunk 4]
```diff
@@ -29,7 +29,7 @@ The **Client ID** and **Client Secret** values of the connection configuration t
 There is no security impact for your environment even though the client secret was logged into the carbon audit logs. However, as an additional precaution, we highly recommend rotating IDP client secrets that are being used in your environment after applying the given patch. The following two approaches can be recommended for IDP secret rotation.
 
 * If the application supports the maintenance of multiple secrets, then update the client secret of the IDP in your environment Then delete the old secret from the application.
-* If the application does not support the maintenance of multiple secrets, then create a new application. Once the new application is created, Then update the client id and client secret of the configured IDP in the environment, and . Then delete the old application.
+* If the application does not support the maintenance of multiple secrets, then create a new application. Once the new application is created, Then update the client id and client secret of the configured IDP in the environment. Then delete the old application.
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2023/WSO2-2022-2357.md:32]:
```suggestion
* If the application does not support the maintenance of multiple secrets, then create a new application. Once the new application is created, then update the client id and client secret of the configured IDP in the environment. Then delete the old application.
```

**Review by 1uffyD9 — CHANGES_REQUESTED** [review]
Suggested few changes.

File: en/mkdocs.yml [hunk 5]
```diff
@@ -79,6 +79,12 @@ nav:
         - 'WSO2-2022-2019': 'security-announcements/security-advisories/2023/WSO2-2022-2019.md'
         - 'WSO2-2022-2043': 'security-announcements/security-advisories/2023/WSO2-2022-2043.md'
         - 'WSO2-2022-2357': 'security-announcements/security-advisories/2023/WSO2-2022-2357.md'
+        - 'WSO2-2022-2023': 'security-announcements/security-advisories/2023/WSO2-2022-2023.md'
+        - 'WSO2-2022-2101': 'security-announcements/security-advisories/2023/WSO2-2022-2101.md'
+        - 'WSO2-2022-2177': 'security-announcements/security-advisories/2023/WSO2-2022-2177.md'
+        - 'WSO2-2022-2182': 'security-announcements/security-advisories/2023/WSO2-2022-2182.md'
+        - 'WSO2-2023-2447': 'security-announcements/security-advisories/2023/WSO2-2023-2447.md'
+        - 'WSO2-2023-2617': 'security-announcements/security-advisories/2023/WSO2-2022-2617.md'
```
Comment (ayomawdb) [inline en/mkdocs.yml:87]: This line should be removed

File: en/docs/security-announcements/security-advisories/2023/WSO2-2022-2182.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
+---
+title: Security Advisory WSO2-2022-2182
+category: security-announcements
+published: June 15, 2023
+version: 1.0.0
+severity: High
+cvss: "CVSS Score:  8.3 (CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:L)"
```
Comment (ayomawdb) [inline en/docs/security-announcements/security-advisories/2023/WSO2-2022-2182.md:7]:
```suggestion
cvss: "8.3 (CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:L)"
```

File: en/docs/security-announcements/security-advisories/2023/WSO2-2022-2177.md [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+severity: Critical
+cvss: "9.4 (CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:L)"
+---
+
+# Security Advisory WSO2-2022-2177
+
+<p class="doc-info">Published: June 15, 2022</p>
+<p class="doc-info">Version: 1.0.0</p>
+<p class="doc-info">Severity: Critical</p>
+<p class="doc-info">9.4 (CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:L)</p>
```
Comment (ayomawdb) [inline en/docs/security-announcements/security-advisories/2023/WSO2-2022-2177.md:15]:
```suggestion
<p class="doc-info">CVSS Score: 9.4 (CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:L)</p>
```

**Review by ayomawdb — APPROVED** [review]

**Review by 1uffyD9 — APPROVED** [review]

**Pull request merged**